packages/@react-native-windows/cli/package.json

```diff
@@ -35,7 +35,8 @@
     "xml-formatter": "^2.4.0",
     "xml-parser": "^1.2.1",
     "xmldom": "^0.5.0",
-    "xpath": "^0.0.27"
+    "xpath": "^0.0.27",
+    "yargs": "^16.2.0"
```

Comment:
I think we can make this a devDepenency since we're injecting the singleton from react-native-windows-init.